pkg/postgres/migrate.go

```diff
@@ -84,6 +85,9 @@ func ResetOnMigration(db *DB, log log15.Logger, doneCh chan struct{}) {
 		for {
 			select {
 			case n, ok := <-listener.Notify:
+				if closed {
+					return
```

Comment:
Is the goal here to drain the notify channel? Does it get closed at the end? Will more than one notification ever be sent after closing the connection?